Source/Core/DolphinNoGUI/PlatformWayland.cpp

```diff
@@ -333,8 +331,8 @@ void PlatformWayland::xdg_toplevel_handle_configure(void* data, struct xdg_tople
   }
   else
   {
-    xdg_surface_set_window_geometry(platform->m_xdg_surface, platform->m_window_x,
-                                    platform->m_window_y, platform->m_window_width,
+    xdg_surface_set_window_geometry(platform->m_xdg_surface, 0,
+                                    0, platform->m_window_width,
```

Comment:
Just a heads up! xdg-shell doesn't actually permit absolute positioning of toplevel windows. It's a deliberate design decision (security issues?). Those x/y coordinates are relative to the wl_surface origin (whose placement is entirely up to the compositor). AFAICT, the only way to position windows with Wayland is to use private protocols for the various compositors.